internal/tools/securitypolicy/helpers/helpers.go

```diff
@@ -68,15 +68,10 @@ func ParseEnvFromImage(img v1.Image) ([]string, error) {
 // be included by default in the security policy.
 // The slice includes only a sandbox pause container.
 func DefaultContainerConfigs() []securitypolicy.ContainerConfig {
-	pause := securitypolicy.NewContainerConfig(
-		"k8s.gcr.io/pause:3.1",
-		[]string{"/pause"},
-		[]securitypolicy.EnvRuleConfig{},
-		securitypolicy.AuthConfig{},
-		"",
-		[]string{},
-		[]securitypolicy.MountConfig{},
-	)
+	pause := securitypolicy.ContainerConfig{
+		ImageName: "k8s.gcr.io/pause:3.1",
+		Command:   []string{"/pause"},
```

Comment:
Should this be locked down further to include the default [pause container environment and that the root fs is readonly](https://github.com/containerd/containerd/blob/main/pkg/cri/server/sandbox_run_linux.go#L48-L49)? (I think [this is the default env](https://github.com/containerd/containerd/blob/main/oci/spec.go#L38)?)